### PR TITLE
test 4

### PR DESCRIPTION
**Pull request by snoack**
This change is [Reviewable](https://reviewable-dev-default-rtdb.firebaseapp.com/reviews/snoack/reviewable-test/7)

### CODE REVIEW

File: bitmask-openvpn.py [hunk 1]
```diff
@@ -176,12 +176,9 @@ def restart_openvpn():
...
-if __name__ == "__main__":
+if  __name__ == "__main__":
     parser = argparse.ArgumentParser()
     parser.add_argument("--log-level", default="WARNING")
     parser.add_argument("--force", action="store_true")
     args = parser.parse_args()
     logging.getLogger().setLevel(args.log_level)
-    check_ca()
-    if update_openvpn_config(args.force) | update_cert(args.force):
-        restart_openvpn()
```
Comment (snoack) [inline bitmask-openvpn.py:187]: comment on last line

Comment (snoack) [reply bitmask-openvpn.py:187]: another one

**Review by snoack — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/7)* status: 0 of 1 files reviewed, 1 unresolved discussion

___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/7#gh-1136066604:-NQWXHMkAgoc0pnXMpUi:bffyzqr) ([raw file](https://github.com/snoack/reviewable-test/blob/164b10643e02562a0567e76e2dd2cbb452686add/bitmask-openvpn.py#L0)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

another one
</blockquote></details>

reply form reviewable

**Review by test-user-96 — COMMENTED** [review]
hey

> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/7#-:-NQlu36u1OA955cKpwRX:b2ni41j)* status: 0 of 1 files reviewed, 2 unresolved discussions

**Review by test-user-96 — COMMENTED** [review]
> *[Reviewable](http://localhost:8080/reviews/snoack/reviewable-test/7)* status: 0 of 1 files reviewed, 2 unresolved discussions (waiting on @snoack)

___
*[`bitmask-openvpn.py` line 0 at r1](http://localhost:8080/reviews/snoack/reviewable-test/7#gh-1136066604:-NR-ZrEa-Oq0jTXQNoKR:bb7p34n) ([raw file](https://github.com/snoack/reviewable-test/blob/164b10643e02562a0567e76e2dd2cbb452686add/bitmask-openvpn.py#L0)):*
<details><summary><i>Previously, snoack (Sebastian Noack) wrote…</i></summary><blockquote>

reply form reviewable
</blockquote></details>

test

**Pull request closed without merging**